cryptography/src/bls12381/dkg/player.rs

```diff
@@ -145,28 +141,18 @@ impl<P: PublicKey, V: Variant> Player<P, V> {
         for (idx, share) in reveals {
             // Verify that commitment is valid
             let commitment = commitments.get(&idx).ok_or(Error::MissingCommitment)?;
-            ops::verify_commitment::<V>(
```

Comment:
In the future if we want to assume commitments are already verified, we can change the argument to `finalize` to be `HashMap<Commitment>` (this makes sense for #1796 where each player runs an arbiter).

Reply:
Actually, an enum here is probably better.